src/clims/services/container.py

```diff
@@ -252,22 +261,37 @@ def __init__(self, app):
     def get_by_name(self, name):
         return self.get(name=name)
 
-    def _search_qs(self, query):
-        # TODO: This is temporary. We will be using elastic for searching.
-        query = query.strip()
+    def filter_from(self, query_builder):
+        query_params = query_builder.parse_query()
+        return self.filter(**query_params)
+
+
+class ContainerQueryBuilder:
```

Comment:
I like that we have a class similar to this. It will encapsulate Elastic stuff later.

Reply:
:)